webui/src/pages/repositories/repository/fileRenderers/types.tsx

```diff
@@ -5,6 +5,7 @@ export interface RendererComponent {
     fileExtension: string | null;
     contentType: string | null;
     sizeBytes: number;
+    presign: boolean | false;
```

Comment:
Just to be aligned, here we use `presign: boolean | false` and in objectViewer we use `presign?: boolean`.
Will it be possible to address the presign the same for components?

Reply:
done